R-package/tests/testthat/test_lgb.plot.tree.R

```diff
@@ -0,0 +1,59 @@
+test_that("lgb.plot.tree works as expected"){
```

Comment:
Please also add tests for the other types of machine learning tasks LightGBM can be used for:

* binary classification
* multiclass classification (where, please note, there are `num_classes` trees produced per iteration)
* learning-to-rank

And for the following model situations:

* uses categorical features

These are all cases that could affect the code as written... for example, categorical features have different splitting rules.